Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/QueueView.js

```diff
@@ -147,41 +149,36 @@ QueueView.prototype = {
         new PopoverTracker(messageElement, this._presentPopoverForPendingCommits.bind(this, latestIterationGetter), queue);
     },
 
-    _popoverLinesForCommitRange: function(trac, branch, firstRevisionNumber, lastRevisionNumber)
+    _popoverLinesForCommitRange: function(commits, branch, commitList)
     {
-        function lineForCommit(trac, commit)
+        function lineForCommit(commits, commit)
         {
             var result = document.createElement("div");
             result.className = "pending-commit";
 
             var linkElement = document.createElement("a");
             linkElement.className = "revision";
-            linkElement.href = trac.revisionURL(commit.revisionNumber);
+            linkElement.href = commits.urlFor(commit.identifier);
             linkElement.target = "_blank";
-            linkElement.textContent = this._formatRevisionForDisplay(commit.revisionNumber, branch.repository);
+            linkElement.textContent = this._formatRevisionForDisplay(commit.identifier, branch.repository);
             result.appendChild(linkElement);
 
             var authorElement = document.createElement("span");
             authorElement.className = "author";
-            authorElement.textContent = commit.author;
+            authorElement.textContent = commit.author ? commit.author.name : '';
             result.appendChild(authorElement);
 
             var titleElement = document.createElement("span");
             titleElement.className = "title";
-            titleElement.innerHTML = commit.title.innerHTML;
+            titleElement.innerHTML = commit.message ? commit.message.split('\n')[0] : '';
             result.appendChild(titleElement);
 
             return result;
         }
 
-        console.assert(trac.indexOfRevision(trac.oldestRecordedRevisionNumber) <= trac.indexOfRevision(firstRevisionNumber));
-
-        // FIXME: To be 100% correct, we should also filter out changes that are ignored by
```

Comment:
I think that this is still an accurate comment.

Reply:
It is still an accurate comment, but I don't see any path to a fix. We would need to duplicate Buildbot's filter logic or have commits.webkit.org own the concept of filtering. I think both are bad ideas, and the benefit is pretty dubious in the first place

Reply:
It wouldn't be too hard to fix actually, just need to post the rules as a JSON somewhere, factoring out of Python code.

Reply:
The rules are dependent on the files that changed, that information is possible to get, but isn't performant to get in the general case